manim/mobject/three_dimensions.py

```diff
@@ -732,6 +791,71 @@ def get_start(self):
     def get_end(self):
         return self.end
 
+    @staticmethod
+    def perp_to_line(point, line, **kwargs):
+        """Returns a line passing through a point and
+        perpendicular to another line.
+
+        Example
+        -------
+
+        .. manim:: PerpendicularLineExample
+            :save_last_frame:
+
+            class PerpendicularLineExample(ThreeDScene):
+                def construct(self):
+                    self.set_camera_orientation(45 * DEGREES, 135 * DEGREES)
+                    ax = ThreeDAxes()
+                    line1 = Line3D([-5, -5, -5], [5, 5, 5], color=RED)
+                    dot = Dot3D([3, 0, 2], color=BLUE)
+                    line2 = Line3D.perp_to_line(dot.get_center(), line1, color=YELLOW)
+                    self.add(ax, line1, dot, line2)
```

Comment:
The angle feels a little too high for this example + its sister method. Maybe bringing to a more central position would look nicer. Also, using `.c2p` to make the lines based on the axes (right now it does nothing 🤔) might also be wise.

![image](https://user-images.githubusercontent.com/70682032/126456603-224abd11-d531-4d23-b95a-c2c4b4056c00.png)